desc/objectives/_stability.py

```diff
@@ -559,8 +559,7 @@ def compute(self, params, constants=None):
         # using the wrong grid
         data = {
             "iota": iota_data["iota"][0] * jnp.ones_like(zeta),
-            "iota_r": iota_data["iota_r"][0] * jnp.ones_like(zeta),
-            "iota_rr": iota_data["iota_rr"][0] * jnp.ones_like(zeta),
+            "shear": iota_data["shear"][0] * jnp.ones_like(zeta),
```

Comment:
Marking to reference

Reply:
Fixed.

Reply:
i see a typo that sets `iota_r` to `iota`